lib/potassium/recipes/api.rb

```diff
@@ -31,4 +43,71 @@ def add_api
       generate "power_api:install"
     end
   end
+
+  def add_graphql
+    gather_gem 'graphql'
+    if get(:authentication)
+      gather_gem 'jwt'
+    end
+    gather_gems(:development, :test) do
+      gather_gem 'graphql_playground-rails'
+    end
+
+    after(:gem_install) do
+      generate "graphql:install --skip_graphiql"
+      playground_route = <<~HEREDOC
+      \n
+        if Rails.env.development?
+          mount GraphqlPlayground::Rails::Engine, at: "/graphiql", graphql_path: "/graphql"
+        end
+      HEREDOC
+      inject_into_file('config/routes.rb',
+        playground_route,
+        after: 'post "/graphql", to: "graphql#execute"'
+      )
```

Comment:
Closing method call brace must be on the same line as the last argument when opening brace is on the same line as the first argument.